.github/workflows/benchmarks.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     runs-on: benchmark
     container:
       image: ubuntu:22.04
-    timeout-minutes: 30
+    timeout-minutes: 60
```

Comment:
is this really necessary?
I thought the benchmarks typically only takes about 10 minutes

Reply:
The changes this week dropped 'normal' runtimes from 28 minutes to 9.

Reply:
ok, given this i'll reduce to 30 again.